src/test/java/com/arangodb/ArangoSearchTest.java

```diff
@@ -266,4 +261,64 @@ public void replaceProperties() {
 		assertThat(link.getFields().iterator().next().getName(), is("value"));
 	}
 
+	@Test
+	public void createGetAndDeleteAnalyzer() {
+		if (!requireVersion(3, 5)) {
+			return;
+		}
+
+		String name = "test-" + UUID.randomUUID().toString();
+		String fullyQualifiedName = db.name() + "::" + name;
+
+		Set<AnalyzerFeature> features = new HashSet<>();
+		features.add(AnalyzerFeature.frequency);
+		features.add(AnalyzerFeature.norm);
+		features.add(AnalyzerFeature.position);
+
+		AnalyzerEntity options = new AnalyzerEntity();
+		options.setFeatures(features);
+		options.setName(name);
+		options.setType(AnalyzerType.delimiter);
```

Comment:
Maybe we could add a loop here and test all available AnalyzerTypes? 
(e.g. if it is somehow combinable - because of the options probably)

Reply:
done